runtime/src/lib.rs

```diff
@@ -203,11 +206,38 @@ parameter_types! {
 	pub const WeightFeeCoefficient: Balance = 1_000;
 }
 
+type NegativeImbalance = <<Runtime as crml_transaction_payment::Trait>::Currency as Currency<
+	<Runtime as frame_system::Trait>::AccountId,
+>>::NegativeImbalance;
+
+pub struct Validators;
+impl OnUnbalanced<NegativeImbalance> for Validators {
+	fn on_nonzero_unbalanced(amount: NegativeImbalance) {
+		let validators = pallet_staking::Module::<Runtime>::current_elected();
+		if validators.len() == 0 {
+			return;
```

Comment:
[thoughts for later] Might still need to handle this case, so that we don't have a permanent imbalance between total issuance and actual issuance

Reply:
Do you think this case would be likely to happen?